app/models/katello/concerns/location_extensions.rb

```diff
@@ -8,6 +8,7 @@ module LocationExtensions
         before_create :set_katello_default
         before_save :refute_katello_default_changed
         before_destroy :deletable?
+        attr_accessible :katello_default
```

Comment:
Look at the output when I tried exactly your changes http://ci.theforeman.org/job/test_katello_pr_core/957/database=postgresql,ruby=1.9.3/console